docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -970,6 +971,24 @@ The values of the `typescript.options.lib` compiler option should be identical t
 *Related configuration file property*: [compilerOptions](configuration-file.md#compileroptions).
 *Related API method*: [runner.compilerOptions](testcafe-api/runner/compileroptions.md)
 
+### --cache
+
+If enabled, TestCafe proxy caches webpage dependencies (like styles and scripts) for webpages visited during tests. When TestCafe revisits the page, it pulls assets from the cache instead of requesting them from the server.
+
+```sh
+testcafe chrome my-tests --cache
+```
+
+TestCafe emulates the native caching mechanism of the browser. For example, in *Chrome* TestCafe only caches resources that *Chrome* itself would cache if run without TestCafe.
+
+TestCafe caches scripts, styles, fonts, and other web assets, except for HTML. TestCafe doesn't cache HTML because that could interfere with [user roles](../guides/advanced-guides/authentication.md#user-roles). Only assets up to **5 MB** in size are stored in cache.
```

Comment:
```suggestion
TestCafe caches scripts, stylesheets, fonts, and other page assets up to **5 MB** in size. TestCafe doesn't cache HTML because that could lead to issues with [user role recall](../guides/advanced-guides/authentication.md#user-roles).
```

Reply:
`Web assets` is a [common term](https://www.google.com/search?newwindow=1&ei=ncQwYK6-Dsqk3APrlpXgAg&q=web+asset&oq=web+asset&gs_lcp=Cgdnd3Mtd2l6EAMyBQgAEJECMgIIADIECAAQQzICCAAyAggAMgIIADIGCAAQFhAeMgYIABAWEB4yBggAEBYQHjIICAAQFhAKEB46BwgAEEcQsAM6CAguEMcBEKMCOgoILhDHARCjAhBDOgQIABAKOgQIABANOgYIABANEB5Q1QxYmhxgpR1oAnACeACAAZABiAGXCJIBAzguM5gBAKABAaoBB2d3cy13aXrIAQjAAQE&sclient=gws-wiz&ved=0ahUKEwjugOGggvjuAhVKEncKHWtLBSwQ4dUDCA0&uact=5). I've barely seen `Page assets` used anywhere.

`up to **5 MB** in size` sounds like we have 5 MB of cache per page. Actually, 5 MB is the size limit for an individual asset. Should ask correctors about this one.

`that could lead to issues with user role **recall**` - I'm not sure it has to do with recall specifically. 
@miherlosev can you confirm?

```suggestion
TestCafe caches scripts, stylesheets, fonts, and other web assets up to **5 MB** in size. TestCafe doesn't cache HTML because that could break [user roles](../guides/advanced-guides/authentication.md#user-roles).
```